test/converters/xcresult3/model3/conversion.go

```diff
@@ -0,0 +1,98 @@
+package model3
```

Comment:
I kept reading it as mode`13` (thirteen), and it took some time for me to realize that it is model`3` (three) 😂